client/src/styles/_bootstrap-variables.scss

```diff
@@ -165,7 +159,7 @@ $spacers: (
 //
 // Settings for the `<body>` element.
 
-$body-bg: #f6f7f8;
+$body-bg: lighten($gray-700, 63);
```

Comment:
is this different from the other `gray-*` colour codes? are we able to use those instead of another variation?
(p.s. I don't mind if this is kept, but could we have the computed colour code commented here as well?)